wp-content/themes/pub/wporg-learn-2024/parts/header.html

```diff
@@ -1,20 +1,6 @@
 <!-- wp:wporg/global-header {"style":{"border":{"bottom":{"color":"var:preset|color|white-opacity-15","style":"solid","width":"1px"}}}} /-->
 
-<!-- wp:wporg/local-navigation-bar {"className":"has-display-contents","backgroundColor":"charcoal-2","style":{"elements":{"link":{"color":{"text":"var:preset|color|white"},":hover":{"color":{"text":"var:preset|color|white"}}}}},"textColor":"white","fontSize":"small"} -->
-
-	<!-- wp:group {"style":{"spacing":{"blockGap":"0"}},"textColor":"light-grey-1","layout":{"type":"flex","flexWrap":"nowrap"}} -->
-	<div class="wp-block-group has-light-grey-1-color has-text-color">
-		<!-- wp:site-title {"level":0,"fontSize":"small","textColor":"white"} /-->
-
-		<!-- wp:query-title {"type":"filter","level":0,"fontSize":"small","fontFamily":"inter","className":"wporg-local-navigation-bar__fade-in-scroll"} /-->
-
-		<!-- wp:post-title {"level":0,"fontSize":"small","fontFamily":"inter","className":"wporg-local-navigation-bar__fade-in-scroll"} /-->
-	</div>
-	<!-- /wp:group -->
-
-	<!-- wp:navigation {"menuSlug":"learn","icon":"menu","overlayBackgroundColor":"charcoal-2","overlayTextColor":"white","layout":{"type":"flex","orientation":"horizontal"},"fontSize":"small"} /-->
-
-<!-- /wp:wporg/local-navigation-bar -->
+<!-- wp:template-part {"slug":"local-nav","className":"has-display-contents","tagName":"div"} /-->
```

Comment:
Now that you've split out the two local-nav partials, could they be merged back into the header partials?

Reply:
Do you mean the change like this: 8e56796, for reducing complexity?